style_static_prefs/Cargo.toml

```diff
@@ -1,7 +1,10 @@
 [package]
-name = "static_prefs"
+name = "stylo_static_prefs"
 version = "0.1.0"
-edition = "2021"
 authors = ["The Servo Project Developers"]
+documentation = "https://docs.rs/stylo_static_prefs/"
+description = "Static configuration crate for Stylo CSS engine"
+repository = "https://github.com/servo/stylo"
+keywords = ["css", "style"]
```

Comment:
These kind of changes belong upstream I think and then we can take them once we integrate them here.